test/index.test.js

```diff
@@ -854,12 +855,31 @@ describe('Optimizely', function() {
 
       it('should send an event for a named page', function() {
         analytics.page('Home');
-        analytics.called(window.optimizely.push, ['trackEvent', 'Viewed Home Page', {}]);
+        analytics.called(window.optimizely.push, ['trackEvent', 'Viewed Home Page', {
+          tags: {
+            name: 'Home',
+            path: '/context.html',
+            referrer: 'http://localhost:9876/?id=' + '<the page id>?',
```

Comment:
@jamesopti ahh yeah you can basically do something like this: https://github.com/segment-integrations/analytics.js-integration-boomtrain/blob/dc1eb52c701a8613ed08c9e05a8e73bde63bd7c8/test/index.test.js#L130-L132 :)